noir-projects/aztec-nr/value-note/src/value_note.nr

```diff
@@ -11,7 +11,9 @@ global VALUE_NOTE_LEN: Field = 3; // 3 plus a header.
 #[aztec(note)]
 struct ValueNote {
     value: Field,
-    owner: AztecAddress,
+    // The nullifying public key hash of the person who owns the note. 
+    // This is used with the app_nullifier_secret_key to ensure that the note can be privately spent.
+    owner_npk_m_hash: Field,
```

Comment:
As @benesjan mentioned somewhere, we don't need the `owner` here it is fairly implicit as well.